packages/components/ng-ovh-order-tracking/src/order-tracking.service.js

```diff
@@ -0,0 +1,65 @@
+import flatten from 'lodash/flatten';
+import map from 'lodash/map';
+import reverse from 'lodash/reverse';
+
+export default class OrderTrackingService {
+  /* @ngInject */
+  constructor($http, $q) {
+    this.$http = $http;
+    this.$q = $q;
+  }
+
+  getOrderDetails(order) {
+    return this.$http
+      .get(`/me/order/${order.orderId}/details`, {
+        headers: {
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
+        },
+      })
+      .then(({ data }) => data);
+  }
+
+  getOrderStatus(order) {
+    return this.$http
+      .get(`/me/order/${order.orderId}/status`)
+      .then(({ data }) => ({
+        ...order,
```

Comment:
Looks like this is doing more than it should, it's not only retrieving the status of the order here